app/views/websites/index.html.erb

```diff
@@ -5,6 +5,17 @@
       <h1>XYPN Scraper</h1>
     </div>
 
+    <div class="col-sm-6 col-sm-offset-3 text-center">
+      <p>
+        The desktop and mobile scores listed are calculated by Google's PageSpeed Insights 
+        and represent a website's overall speed and performance. PageSpeed Insights analyzes 
+        a page to see if it follows Google's recommendations for making a page render in under a 
+        second. Research has shown that any delay longer than a second will cause the user to 
+        interrupt their flow of thought, creating a poor experience. A higher score is better 
+        and a score of 85 or above indicates that the page is performing well.
+      </p>
+    </div>
```

Comment:
Last minute addition: can you add this link? 

https://www.doubleclickbygoogle.com/articles/mobile-speed-matters/